cmd/storage-class.go

```diff
@@ -149,11 +149,13 @@ func validateSSParity(ssParity, rrsParity int) (err error) {
 
 // Returns the data and parity drive count based on storage class
 // If storage class is set using the env vars MINIO_STORAGE_CLASS_RRS and MINIO_STORAGE_CLASS_STANDARD
+// or config.json fields
 // -- corresponding values are returned
-// If storage class is not set using environment variables, default values are returned
+// If storage class is not set during startup, default values are returned
 // -- Default for Reduced Redundancy Storage class is, parity = 2 and data = N-Parity
 // -- Default for Standard Storage class is, parity = N/2, data = N/2
-// If storage class is not present in metadata, default value is data = N/2, parity = N/2
+// If storage class is not present in metadata
```

Comment:
Suggestion: Since we are not passing the metadata to this function, it would be clear to say "If storage class is empty ..." (instead of not present in metadata).

Reply:
done